services/users/service.go

```diff
@@ -0,0 +1,66 @@
+package users
+
+import (
+	"user-curd/models"
+	"user-curd/services"
+	"user-curd/stores"
+	"errors"
+)
+
+type UserServiceHandler struct {
+	stores stores.Store //Users interface
+}
+
+func New(store stores.Store) services.Services {
+	return UserServiceHandler{stores: store}
+}
+
+func (u UserServiceHandler) EmailValidation(email string) (bool, error) {
```

Comment:
need not export these helper functions. Also try to keep helper functions in a separate file